src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/ProcessedEnvelopeNotifier.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.QueueClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.ProcessedEnvelope;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+
+/**
+ * Notifies about successfully processed envelopes via queue.
+ */
+@Service
+@Profile("!nosb") // do not register for the nosb (test) profile
+public class ProcessedEnvelopeNotifier implements IProcessedEnvelopeNotifier {
+
+    private final Logger log = LoggerFactory.getLogger(ProcessedEnvelopeNotifier.class);
+
+    private final QueueClient queueClient;
+    private final ObjectMapper objectMapper;
+
+    public ProcessedEnvelopeNotifier(
+        @Qualifier("processed-envelopes") QueueClient queueClient,
+        ObjectMapper objectMapper
+    ) {
+        this.queueClient = queueClient;
+        this.objectMapper = objectMapper;
+    }
+
+    public void notify(String envelopeId) {
+        try {
+            String messageBody =
+                objectMapper.writeValueAsString(new ProcessedEnvelope(envelopeId));
```

Comment:
Do we really need a ProcessedEnvelope object ?
we can just do ```objectMapper.writeValueAsString(ImmutableMap.of("id",envelopeId));```

Reply:
ProcessedEnvelope defines the format of the queue message (even though it's very simple). This is consistent with how we define it for the incoming messages. If we replaced it with a map, that would also mean it's the notifier that defines the format now. I think it's slightly clearer as it it.